### ISSUE TITLE
Tooltip accessibility

### ISSUE THREAD

**Issue by alyce** [magda-io/magda#2548]
As identified, our current help tooltip is not keyboard accessible, which means we fail our accessibility compliance. 

I just did a quick google, there is plenty of stuff online about accessible tooltips, such as [this page](https://van11y.net/accessible-simple-tooltip/)

I suspect we'll need to make the ? icon focusable so we can let it have keyboard accessibility but this probably needs a bit more exploration
